EMS/core-bundle/src/Controller/Component/MediaLibraryController.php

```diff
@@ -170,20 +170,23 @@ public function deleteFolder(Request $request, UserInterface $user, string $fold
 
     public function getFiles(Request $request): JsonResponse
     {
+        $query = $request->query;
+
         $folderId = $request->get('folderId');
         $folder = $folderId ? $this->mediaLibraryService->getFolder($folderId) : null;
 
-        $sortOrder = $request->get('sortOrder');
+        $sortOrder = $query->get('sortOrder');
```

Comment:
Is it always defined? Maybe set a default value?

Reply:
yes the default is null, ```public function get(string $key, $default = null)```